pom.xml

```diff
@@ -177,12 +177,18 @@
 		</dependency>
 
 		<dependency>
-			<groupId>commons-httpclient</groupId>
-			<artifactId>commons-httpclient</artifactId>
-			<version>3.1</version>
+			<groupId>org.apache.httpcomponents</groupId>
+			<artifactId>httpclient</artifactId>
+			<version>4.5.9</version>
```

Comment:
We should use this version:
https://mvnrepository.com/artifact/org.apache.httpcomponents/httpclient/4.5.13